docker/run-mysql.sh

```diff
@@ -24,7 +24,7 @@ start() {
     docker run --rm \
                --link $CONTAINER:mysql \
                -v $PWD/docker/sql_validate.sh:/working/sql_validate.sh \
-               mysql:$MYSQL_VERSION \
+               mysql/mysql-server:$MYSQL_VERSION \
```

Comment:
what does this change do?

Reply:
ah, sorry, that is a mistake.  Will revert it back. 
(I am using Apple Chip mac and it does not support Mysql 5. So as a workaround, I change here to mariaDb, forgot to clean it up.)